packages/replay/src/replay.ts

```diff
@@ -915,77 +917,40 @@ export class ReplayContainer implements ReplayContainerInterface {
 
     const currentTimestamp = new Date().getTime();
 
-    const sdkInfo = {
-      name: 'sentry.javascript.integration.replay',
-      version: __SENTRY_REPLAY_VERSION__,
-    };
+    const hub = getCurrentHub();
+    const client = hub.getClient();
+    const scope = hub.getScope();
+    const transport = client?.getTransport();
```

Comment:
l: let's avoid optional chaining

Reply:
In the ES6 codebase, is this actually "bad"? 🤔

Reply:
Hmm, so we still polyfill this when transpiling because optional chaining [was added in es2020](https://dev.to/diraskjavascript/how-to-use-optional-chaining-in-javascript-es2020-51bm). We can definitely revisit this though so feel free to ignore my comment